precache.php

```diff
@@ -69,6 +69,8 @@ public function onShutdown()
 
                 try {
                     $page = $pages->get($path);
+                    $this->grav['page'] = $page;
```

Comment:
Needs to be:

```php
unset($this->grav['page']);
$this->grav['page'] = $page;
```

This is because of container will lock the instance once someone tries to get it.

Reply:
```suggestion
                    unset($this->grav['page']);
                    $this->grav['page'] = $page;
```

Same comment but as a suggestion in order to apply it

Reply:
Applied!